CHANGES.md

```diff
@@ -1,3 +1,17 @@
+## Version 0.6.1
```

Comment:
doesn't this change the name of many implicits? Should we mention that and mention that it is not strictly binary compatible with `0.6.1` due to that change?

Also, why not `0.7.0` to signify that change rather than `0.6.1`?